index.xml

```diff
@@ -11163,10 +11183,8 @@ partial interface Window {
                     <li>Let <var>desc</var> be a newly created <a>Property Descriptor</a> with no fields.</li>
                     <li>Set <var>desc</var>.<span class='prop'>[[Value]]</span> to the result of <a class='dfnref' href='#dfn-convert-idl-to-ecmascript-value'>converting</a>
                       <var>value</var> to an ECMAScript value.</li>
-                    <li>If the named property is defined to be <a class='dfnref' href='#dfn-unenumerable'>unenumerable</a>,
-                      then set <var>desc</var>.<span class='prop'>[[Enumerable]]</span> to <span class='esvalue'>false</span>,
-                      otherwise set it to <span class='esvalue'>true</span>.</li>
-                    <li>Set <var>desc</var>.<span class='prop'>[[Writable]]</span> to <span class='esvalue'>true</span> and
+                    <li>Set <var>desc</var>.<span class='prop'>[[Writable]]</span> to <span class='esvalue'>true</span>,
```

Comment:
Well, then setting `[[Enumerable]]` to true unconditionally would be wrong too, right?

But I _think_ this is "4.6.5.1. Named properties object [[GetOwnProperty]] method", not 4.8.2.